src/three-components/SmoothControls.ts

```diff
@@ -346,6 +355,11 @@ export class SmoothControls extends EventDispatcher {
     return true;
   }
 
+  setRadius(radius: number) {
```

Comment:
Yeah, I wasn’t thinking of any of this as public. I’ll take another pass over it.

Reply:
:+1: for your reference: in many of our Three.js "components", access is not very well hashed out. But, for anything converted to TypeScript, members are implicitly `public` unless declared otherwise.

Reply:
Ah, that's very helpful to know. You might want to put that in the wiki. At the moment all it has is a link to the LitElement documentation, but that page doesn't seem to say anything about what public or private is. I had been thinking things were implicitly private unless exported.

Reply:
Noted in the doc, thanks for the feedback!